helix-view/src/editor.rs

```diff
@@ -153,7 +154,8 @@ impl Editor {
 
         Self {
             tree: Tree::new(area),
-            next_document_id: 0,
+            // Safety: 1 is non-zero
+            next_document_id: unsafe { NonZeroUsize::new_unchecked(1) },
```

Comment:
Call `DocumentId::default()` here, store `DocumentId` instead of `NonZeroUsize` as `next_document_id`.

Reply:
Done